en/navigation_refresh.ftl

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+navigation-refresh-mozilla = { -brand-name-mozilla }
+
+## About us
+
+navigation-refresh-about-us = About us
+navigation-refresh-close-about-us-menu = Close About us menu
+navigation-refresh-mozilla-manifesto = { -brand-name-mozilla } manifesto
+navigation-refresh-mozilla-foundation = { -brand-name-mozilla-foundation }
+navigation-refresh-get-involved = Get involved
+navigation-refresh-innovation-projects = Innovation Projects
```

Comment:
Should this be `Innovation projects` (lower case P) to be consistent with other list of items?

Reply:
Sentance case is the official word going foward.

Reply:
People are doing a bit of everything right now. @craigcook do you want take a pass on capitalization on Monday? I can if you're busy but I know you're better with speling.

Reply:
Let's get this updated soon as it's time sensitive. Anymore strings to come in the next couple of days?

Reply:
@alexgibson not sure if you can help fix this. We are pressing for time. If this is the last update before next week, let's finalize it. If there are more things to come, can we bundle them? Thank you!

Reply:
@peiying2 I will take a look here thanks for the ping

Reply:
@peiying2 PR here that should hopefully address things: https://github.com/mozilla/bedrock/pull/15164